plugins/azure-devops/src/components/BuildTable/BuildTable.tsx

```diff
@@ -149,7 +149,10 @@ const columns: TableColumn[] = [
     field: 'queueTime',
     width: 'auto',
     render: (row: Partial<RepoBuild>) =>
-      DateTime.fromJSDate(row.queueTime ?? new Date()).toRelative(),
+      (row.queueTime
+        ? DateTime.fromISO(row.queueTime)
+        : DateTime.now()
+      ).toRelative(),
```

Comment:
Might be worth looking at this ADR for when displaying dates in the frontend to get them all standardized. https://backstage.io/docs/architecture-decisions/adrs-adr012

Reply:
Don't think this covers relative times such as `10 seconds ago` but thanks for sharing the ADR 😄

Reply:
Thanks for pointing out the ADR, will keep that in mind, in this case as Marley mentioned we are using relative times so that way you can quickly see how old the builds are

Reply:
ahha - ok I see it now.